app/controllers/patients_controller.rb

```diff
@@ -324,18 +334,21 @@ def update_status
         next unless params[:apply_to_group_cm_only_date].present?
 
         # turn off continuous exposure if LDE is updated
-        lde_date = params.permit(:apply_to_group_cm_only_date)[:apply_to_group_cm_only_date]
-        if member[:continuous_exposure]
-          History.monitoring_change(patient: member, created_by: 'Sara Alert System', comment: 'System turned off continuous exposure because monitoree is no
-          longer being exposed to a case.')
-        end
-        member.update(last_date_of_exposure: lde_date, continuous_exposure: false)
+        update_lde(member, params[:apply_to_group_cm_only_date])
       end
     else # update patient
       update_fields(patient, params, :patient, :none)
     end
   end
 
+  def update_lde(patient, lde_date)
```

Comment:
Always a fan of breaking this out 👍

Reply:
ended up having to remove this due to History comment changes